### ISSUE TITLE
1.18.2 Crash (Registry is already frozen)

### ISSUE THREAD

**Issue by North-West-Wind** [gamma-delta/HexMod#33]
Forge: 40.0.31
Minecraft: 1.18.2
Mods: Hex Casting, Patchouli, Kotlin for Forge
[Latest.log](https://pastebin.com/4RnyH5y2) | [Crash report](https://pastebin.com/wirCsPgT)

This mod seems really fun but I cannot play it :(

**Comment by gamma-delta** [comment]
Yep that's a known issue, it's fixed in 0.7.0... sorry! (It'll probably be out in less than a week).

For now, 0.6.0 works on 1.18.1.
